### ISSUE TITLE
Feature: Have a means to configure aemfed, browsersync, etc.

### ISSUE THREAD

**Issue by chamberlainpi** [abmaonline/aemfed#30]
Either by a file specified via `aemfed --config ./aemfed.config.js`, or a new entry in `package.json` (ie: `"aemfed": { ... }`)

I'm currently having issues with some URL requests to Web Fonts returning HTTP 200 with headers containing "**connection: close**", resulting blank Font files for some reason.

It would be nice to have greater control over the underlying options of **aemfed** & especially **browsersync** to inspect the requests / troubleshoot the proxy.

I'm not even sure, but maybe **browsersync** already looks for such a config if present in the root project directory?

**Comment by chamberlainpi** [comment]
For now I've been using a workaround in Chrome, via an extension called **Requestly**.

https://chrome.google.com/webstore/detail/requestly-redirect-url-mo/mdnleldcmiljblolnjhpnblkcekpdkpa

And **CORS-Unblock**:
https://chrome.google.com/webstore/detail/cors-unblock/lfhmikememgdcahcdlaciloancbhjino/

By adding the following rules, I've been able to make the web requests *behave* like they were served directly from the port :4502 (the actual AEM author instance on my local dev environment) 

1) **Modify Headers Rule:** _Referer_, _Host_, _Origin_ = Match them the same as how your original :4502 requests gets sent. Don't forget to set a "If URL contains..." path that matches only for where your resources are located.
 
2) **Modify Headers Rule:** _Replace_, _With_ = Set this to whatever you use for your `localhost:3000` VS. `localhost:4502`. Again, don't forget to set a "If URL contains..." path that matches only for where your resources are located.

And that should do it!
